Twitter_Sentiment/projects/twittersentiment_project_1.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import json as json
+import numpy as np
+
+def main():
+    print("Hello! Today we are going to try to classify an expression's sentiment with a Machine Learning model!")
+
+    url = input("What is your endpoint URL?\n")
+    url = url.strip()
+
+    expression = input("What's on your mind?\n")
+    data ={"Expression": expression}
+
+    get_prediction(url, data)
```

Comment:
label=get_prediction(url, data)

Please use the label value and add if statement to print the sentiment